lib/tasks/npm-task.js

```diff
@@ -13,47 +20,86 @@ class NpmTask extends Task {
     // The command to run: can be 'install' or 'uninstall'
     this.command = '';
 
-    this.npm = this.npm || require('../utilities/npm');
+    this.versionConstraints = '3 || 4';
```

Comment:
would it make sense to turn `versionConstraints` into a `const` on the top of the file or there was some reasoning behind having it as an instance property?